.gitignore

```diff
@@ -1,2 +1 @@
-.DS_Store
-
+.DS_Stores
```

Comment:
@semenetsoleksandr , it was correct to use `.DS_Store` for MacOS, and please add `Thumbs.db` this is equivalent for Windows. Add popular editors’ cache directories. For WebStorm - `.idea`, VSCode.

Reply:
update .gitignore file